packages/hardhat-core/README.md

```diff
@@ -42,6 +42,6 @@ Go to [CONTRIBUTING.md](./CONTRIBUTING.md) to learn about how to set up Hardhat'
 
 [Follow Hardhat on Twitter.](https://twitter.com/HardhatHQ)
 
-## Happy building!
+## Happy buidling!
```

Comment:
```suggestion
## Happy #BUIDLing!
```
This helps identify it's not a misspelling for future BUIDLers here.
For canonical usage, see https://ethdenver.com :pray:

Reply:
I think this will definitely help newcomers understand that this is intentional 👍